### DISCUSSION TITLE
For the example of teleport, does teleport not connect after that

### DISCUSSION THREAD

**Discussion by dengbupapapa** [bcakmakoglu/vue-flow#1431] [Q&A]
For the example of teleport, does teleport not connect after that？（[https://vueflow.dev/examples/teleport.html](https://vueflow.dev/examples/teleport.html)）

I actually want to create a node outside the canvas that can then connect to a node inside the canvas, is that possible?

**Comment by bcakmakoglu** [comment]
No that's not possible with built-in features because the edges are drawn in an svg container that lives *inside* of the viewport (the "background" of the graph if you will).

**Comment by dengbupapapa** [comment]
But what I see in the video in this issue is the ability to connect nodes outside of the canvas—— #174

**Reply by dengbupapapa** [reply]
Or how can I fix a node in the canvas?

**Reply by dengbupapapa** [reply]
> But what I see in the video in this issue is the ability to connect nodes outside of the canvas—— #174
Sorry, I read it wrong, but is there a way to pin a node to a location in the window without having to move with the canvas

**Reply by bcakmakoglu** [reply]
Nope, no built-in feature that would allow you to do that.
You could try adding a "node" into the Panel but again, drawing edges won't properly work for that since the coordinates wouldn't be correct when the pane is moved around and the node was "static" in it's position.

You'd really have to implement your own svg layer, draw an edge yourself and keep updating it when it's connected node moves around. In other words completely re-implement the edge logic that VueFlow provides for a "static" node.
